cleanlab/count.py

```diff
@@ -87,7 +87,7 @@ def num_label_issues(
       Array of estimated class label error statisics used for identifying label issues,
       in same format expected by :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` function.
       The `confident_joint` can be computed using :py:func:`count.compute_confident_joint <cleanlab.count.compute_confident_joint>`.
-      If not provided, it is internally computed from the given (noisy) `labels` and `pred_probs`.
+      It is internally computed from the given (noisy) `labels` and `pred_probs`.
```

Comment:
Should we add another `estimation_method` option in num_label_issues() such that `confident_joint` when provided by user can at least be used for something?